### PR TITLE
BCN - Dafne

### PR DESCRIPTION
**Pull request by dafneolca**
BCN - Dafne

### CODE REVIEW

**Comment by dafneolca** [comment]
BCN - Dafne

File: starter-code/index.html [hunk 1]
```diff
@@ -1,22 +1,52 @@
...
+      </div>
+    </div>
+  </div>
+
+
+
+</div>
+
+</body>
+<footer>
```
Comment (tawebbcn) [inline starter-code/index.html]: Better use a list for "About Us", "Support"...

File: starter-code/css/main.css [hunk 2]
```diff
@@ -7,4 +7,59 @@ light blue: #1c5380
...
+.top-block{
+  background-color: #06365f;
+
+  background-size: cover;
+  background-position: top;
+  height: 500px;
+
+}
+
+h2 {
```
Comment (tawebbcn) [inline starter-code/css/main.css]: Try to avoid type selectors (h2, footer...).

File: starter-code/index.html [hunk 3]
```diff
@@ -1,22 +1,52 @@
...
-  Capture and Share the World's Moments
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+<div class="main">
+  <div class="iphone">
+    <img src="images/phones.png">
+  </div>
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+  <div class="textcontent">
```
Comment (tawebbcn) [inline starter-code/index.html]: Don't abuse of divs, there are more semantical tags.

File: starter-code/css/main.css [hunk 4]
```diff
@@ -7,4 +7,59 @@ light blue: #1c5380
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  background-color: #1c5380;
```
Comment (tawebbcn) [inline starter-code/css/main.css]: You have an image for the background.

**Pull request closed without merging**